apps/condo/domains/marketplace/components/MarketplaceOnboarding/RequisitesSetup.tsx

```diff
@@ -139,7 +147,7 @@ export const RequisitesSetup: React.FC = () => {
                 bic: get(acquiringContext, ['invoiceRecipient', 'bic'], ''),
                 account: get(acquiringContext, ['invoiceRecipient', 'bankAccount'], ''),
                 taxType: get(acquiringContext, 'invoiceTaxRegime'),
-                taxPercent: get(acquiringContext, 'invoiceVatPercent'),
+                taxPercent: Number(get(acquiringContext, 'invoiceVatPercent', '')).toString(),
```

Comment:
It seems that a type error will occur when creating “without VAT”.
<img width="1679" alt="image" src="https://github.com/open-condo-software/condo/assets/64303474/a6a1676d-ac0f-435f-99e1-9f596e5d09b6">

Reply:
empty string invalid value for Decimal type